spec/controllers/gardens_controller_spec.rb

```diff
@@ -12,11 +12,72 @@
 
 require 'rails_helper'
 
-describe GardensController do
-  login_member
+RSpec.describe GardensController, type: :controller do
+  include Devise::Test::ControllerHelpers
+  let(:valid_params) { { name: 'My second Garden' } }
 
-  def valid_attributes
-    member = FactoryGirl.create(:member)
-    { name: 'My Garden', owner_id: member.id }
+  context "when not signed in" do
+    let(:garden) { double('garden') }
+    describe 'GET new' do
+      before { get :new, id: garden.to_param }
```

Comment:
i use `before` so that all the asserts happen -- it means when you break something, you get more clues on the first failed test run. it will say which are failing, instead of just the first to fail in an `it` block. 
e.g.
```
describe "METHOD endpoint" do
   before { make request }
   it { assert something }
   it { assert more }
   it { assert yet more }
end
```

instead of 
```
describe "METHOD endpoint" do
   it do
     make request
     assert something
     assert more
     assert yet more
  end
end
```